massgov/mayflower
# Patternlab/DP-1379  add focus styles

## Description
Add focus states to match hover states to header search and topic pages

## Related Issue / Ticket

- [DP-1379](https://jira.mass.gov/browse/DP-1379)

## Steps to Test


1. Open the topics page: http://localhost:3000/patterns/05-pages-topic/05-pages-topic.html
Tab focus to the header search form and button. The button focus now matches the hover state.
Tab through the page to see that the decorated links and section links have focus states that match the hover states.

## Screenshots
![Screen Shot 2019-04-04 at 3 08 49 PM](https://user-images.githubusercontent.com/18662734/55585341-0ce33c00-56ec-11e9-9e2a-779033853f3b.png)
![Screen Shot 2019-04-04 at 3 09 03 PM](https://user-images.githubusercontent.com/18662734/55585356-12408680-56ec-11e9-85c9-41ba430d2c40.png)
![Screen Shot 2019-04-04 at 3 09 12 PM](https://user-images.githubusercontent.com/18662734/55585362-153b7700-56ec-11e9-9773-cc287890a98f.png)
